utils/mwm.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+import os
+import subprocess
+from string import Template
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class OSMToMWM(object):
+    """
+    Thin wrapper around osmconvert to convert osm xml to mwm.
+    """
+
+    def __init__(self, osm=None, pbffile=None, mwmfile=None, debug=False):
+        self.osm = osm
+        if not os.path.exists(self.osm):
+            raise IOError('Cannot find raw OSM data for this task.')
+        self.pbffile = pbffile
+        self.mwmfile = mwmfile
+        if not self.pbffile:
+            # create pbf path from osm path.
+            root = self.osm.split('.')[0]
+            self.pbffile = root + '.pbf' 
+        if not self.mwmfile:
+            # create mwm path from osm path.
+            root = self.osm.split('.')[0]
+            self.mwmfile = root + '.mwm' 
+        self.debug = debug
+        mwm_root_path = settings.EXPORT_MWM_ROOT
+        self.cmd = Template('osmconvert $osm --out-pbf >$pbf')
```

Comment:
Aren't pbf files generated somewhere in the exporting process already? I guess you can specify this dependency and use a pre-generated pbf.